Dockerfile

```diff
@@ -1,12 +1,11 @@
 # Compiler container
-FROM node:12-alpine AS compiler
+FROM node:16-alpine AS compiler
 RUN apk add --no-cache build-base git bash
 WORKDIR /usr/src/app
-COPY package.json ./
-RUN npm i --cache /tmp/1
-COPY . ./
+COPY / ./
```

Comment:
Shouldn't be . ./

Reply:
And in this scenario we need to do the npm install and build it inside the container. 

`
COPY . ./
RUN npm i
`

Since in our prepare script its build there is no need for the build step.

Reply:
Done